.travis.yml

```diff
@@ -1,22 +1,21 @@
 osx_image: xcode10.1
 jobs:
   include:
-    - stage: test Linux
+    - stage: Run tests
       script: ./Scripts/travis/test-Linux.sh
       os: linux
       dist: xenial
-    - stage: test macOS
+    - os: osx
       script: ./Scripts/travis/test-macOS.sh
-      os: osx
     - stage: release new version
       script: ./Scripts/travis/new-release.sh
       os: osx
-      if: branch = master
+      if: branch = master AND type = push AND fork = false
```

Comment:
Will this prevent release if an external user does a PR for a new feature/fix?

Reply:
Yes it basically requires it to be considered a "push" to the master branch from a non fork repo. So that would only be the case when we recently merged something onto the master branch.